components/consumers/pdf/main.go

```diff
@@ -140,6 +147,11 @@ func buildPdf(data any, pw playwright.Wrapper) (string, []byte, error) {
 		return "", nil, fmt.Errorf("could not generate pdf from page %s, err: %w", reportPage, err)
 
 	}
+
+	// delete the intermediate HTML file
+	if err := os.Remove(reportHTMLPath); err != nil {
```

Comment:
⛏️ I think we should log this error and keep going as it's not critical - more or less like the other check that you added to close the file 🤓